### ISSUE TITLE
Make a regressor for sentiment based on AllenNLP tool

### ISSUE THREAD

**Issue by AlexDel** [AlexDel/levheimcube#10]
We have an example for lstm-based classifier which predicts emotion label. We plan to rework it as a regressor that works with another dataset and predicts coordinates of the cube

Use the following code as a reference.
https://github.com/AlexDel/levheimcube/blob/master/projects/allennlp/lstm_simple_classifier.py
https://github.com/AlexDel/levheimcube/blob/master/projects/allennlp/tools/vk_data_loader.py

Use this dataset for your project - https://storage.yandexcloud.net/nlp-dataset-bucket-1/toloka-vk-proceedings-2020/toloka_recalc_lean.csv

Your tasks:
- Split the dataset into train and validation parts.
- Change the data loader to take (dop, nor, ser) fields from the data.
- Change classification layer (torch.nn.Linear) to regression layer to predict 3 values (dop, nor, ser) from dataset
- Use MeanAbsoluteError as loss function and metric
- Experiment with other losses (torch.nn.MSELoss. torch.nn.SmoothL1Loss), keaping MAE as a target metric. Report the best loss fn.
- Share your results as a link to Jupyter Notebook.

I suggest you to use use Colab Notebooks or Yandex Datasphere notebooks, as it provides persistence and V100 GPU for free,

Estimation: 40 hours
